.github/CODEOWNERS

```diff
@@ -305,6 +305,7 @@
 /core/templates/services/questions-list.service*.ts @kevintab95
 /core/templates/components/question-directives/ @aks681 @kevintab95
 
+
```

Comment:
Check "Files Changed" before asking for review, please remove any changes that aren't needed.

Reply:
Hi @seanlip, I've added this change while resolving some conflicts because I noticed that we use 2 new lines after each project (see above).

Reply:
Should I undo this or leave it as-is?

Reply:
Ah sorry, didn't realize it was a general convention. OK to leave as-is. Thanks!